src/components/AppNavigation/Settings/SettingsImportContacts.vue

```diff
@@ -338,6 +338,10 @@ export default {
 			background-color: transparent;
 		}
 	}
+	// this is for the import button on emptycontent, because it doest fit the style of new contact
+	::v-deep .button-vue__icon {
+		margin-left: 19px;
+	}
```

Comment:
This puts the text off center. I'd prefer to have it centered.

Reply:
this was an attempt to fix the icon and the text, see my second screenshot on desc. If i dont add it, the icon and the text is off, also the new contact button is slightly bigger, i dont understand why. (screenshot on my comment)